mongoid-observers.gemspec

```diff
@@ -18,7 +18,6 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rails-observers", "~> 0.1.2"
```

Comment:
This gem entirely depends on `rails-observers`. Remove this line, the gem user needs to add this gem manually in their Gemfile.